restapi/src/main/java/io/stargate/web/docsapi/service/query/rules/ExpressionUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.web.docsapi.service.query.rules;
+
+import com.bpodgursky.jbool_expressions.Expression;
+import com.bpodgursky.jbool_expressions.rules.RuleSet;
+import io.stargate.web.docsapi.service.query.FilterExpression;
+
+public class ExpressionUtils {
+
+  public static Expression<FilterExpression> toSimplifiedCnf(
```

Comment:
small java doc please to explain what's this for, especially because method name does not specify anything about negating..

Reply:
Javadoc added